### PR TITLE
Adding learn git guide from Write the Docs unconference session

### PR DESCRIPTION
**Pull request by juliakm**
Co-authored-by: Mike Lewis <mlewis@gitlab.com>
Co-authored-by: Achilleas Pipinellis <axilleas@users.noreply.github.com>
Co-authored-by: Alexander Dade <elegantechesterr@gmail.com>

### CODE REVIEW

**Comment by plaindocs** [comment]
Hi Folks, this looks like good stuff!

The checks are currently failing due to capitalization of `The` in the Write the Docs:

 guide/learn-git-contribute.rst
 1:26    error  Use 'Write the Docs' instead    WTD.Branding 
                of 'Write The Docs'                          
 31:16   error  Use 'Write the Docs' instead    WTD.Branding 
                of 'Write The Docs'                          
 124:45  error  Use 'Write the Docs' instead    WTD.Branding 
                of 'Write The Docs'

**Comment by juliakm** [comment]
@plaindocs Thanks for the help! This is ready for another review if you have time.

**Comment by ericholscher** [comment]
@Geodun You might find this useful to integrate into the upcoming Contribution section.

File: docs/guide/learn-git-contribute.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Prerequisites
+-------------
+
+1. `Create a GitHub account`_.
+2. `Download and install Git`_.
+3. Open a terminal window and follow the instructions to `associate your
+   GitHub username with your local Git installation`_.
+
+   1. In macOS: Open the **Terminal**\ app.
```
Comment (plaindocs) [inline docs/guide/learn-git-contribute.rst:25]: These should probably be bullets, as you do one or the other.

File: docs/guide/learn-git-contribute.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+   2. In Windows: From the Start Menu, open **Git Bash**.
+
+Start using Git and modify the source file of a page
+----------------------------------------------------
+
+1.  Visit the `Write the Docs www project`_.
+
+2.  Click the **Fork**\ button in the upper-right corner to create a
```
Comment (plaindocs) [inline docs/guide/learn-git-contribute.rst:33]:
```suggestion
2.  Click the **Fork** button in the upper-right corner to create a
```

File: docs/guide/learn-git-contribute.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    copy of the project in your GitHub account. The new page for the
+    forked project opens.
+
+3.  Click the **Clone or download** button and copy the https URL from
+    the project page.
+
+4.  Open a terminal window so that you can run ``git`` commands.
+
+    1. In macOS: Open the **Terminal** app.
```
Comment (plaindocs) [inline docs/guide/learn-git-contribute.rst:42]: Bullets again I think

**Review by plaindocs — COMMENTED** [review]
Couple of minor nits, and the last thing would be to include it in a suitable `toctree`. Maybe under `Tools of the Trade` around line 72 of `guide/index`? What do you think.

Another good place to link it from would be `guide/contributing.rst`

After that I'd love to merge this in. Thank you!

**Review by plaindocs — APPROVED** [review]

**Pull request merged**